analytics_data_api/v0/tests/views/test_learners.py

```diff
@@ -625,11 +625,20 @@ def _get_full_engagement_ranges(self):
             normal_floor = 800.8
             G(ModuleEngagementMetricRanges, course_id=self.course_id, start_date=start_date, end_date=end_date,
               metric=metric_type, range_type='normal', low_value=normal_floor, high_value=max_value)
+
             expected['engagement_ranges'][metric_type] = {
-                'below_average': [0.0, low_ceil],
-                'average': [normal_floor, max_value],
-                'above_average': None
+                'class_rank_average': [normal_floor, max_value],
```

Comment:
So `'class_rank_bottom': [0.0, low_ceil]` is implied here? How would a client know it's that and not just omitted because it's not defined?